Web/Edubase.Web.UI/Assets/Scripts/GiasSearchForm/GiasRadioToggles.js

```diff
@@ -90,4 +103,13 @@ $.fn.giasRadioToggles = function (opts) {
   });
 };
 
+$(document).ready(function() {
+  const radioToggleInstance = $('#results-container').giasRadioToggles();
+
+  $('#filter-apply').on('click', function(e) {
+    e.preventDefault();
+    radioToggleInstance.data('giasRadioToggles').applyFilters();
+  });
+});
+
```

Comment:
It's not clear to me which page this is on but I _think_ it's active on the search pages.

The search pages seem to work well, therefore ✅